mgmtsystem/azure.py

```diff
@@ -66,8 +67,9 @@ def pre_script(self):
         $myazurename = "{}"
         $myazurepwd = ConvertTo-SecureString "{}" -AsPlainText -Force
         $azcreds = New-Object System.Management.Automation.PSCredential ($myazurename, $myazurepwd)
-        Login-AzureRMAccount -Credential $azcreds
-        Get-AzureRmSubscription -SubscriptionId \"{}\" -TenantId \"{}\" | Select-AzureRmSubscription
+        Login-AzureRMAccount -Credential $azcreds |Out-Null
```

Comment:
I like Out-Null  Good addition.  Make sure you add a space after the |  there are several places in this script like that so check them all